sb-utils/src/main/resources/cassandra.config.properties

```diff
@@ -0,0 +1,13 @@
+coreConnectionsPerHostForLocal=4
```

Comment:
@Hari-stackroute - where are we using these properties?

Reply:
@vrayulu : It is not newly added, just replaced the file locations. If the properties are there in environment, it can be read from the properties.
Question is we also didn't tried testing by removing such property files.

Reply:
@Hari-stackroute - same comment as the last one: new service and repository is an opportunity to do things in the right way. 
If you replaced the file locations, it should have been removed from the old location. But I dont see that changes in list of files for this PR? In fact, "cassandra.config.properties" is present in 4 locations in the repo - https://github.com/Hari-stackroute/groups-service/tree/SH-615.